static/app/views/organizationIntegrations/integrationRepos.tsx

```diff
@@ -260,16 +247,6 @@ class IntegrationRepos extends AsyncComponent<Props, State> {
 
 export default withOrganization(IntegrationRepos);
 
-const HeaderText = styled('div')`
-  padding-left: ${space(2)};
-  flex: 1;
-`;
-
-const DropdownWrapper = styled('div')`
-  padding-right: ${space(1)};
-  text-transform: none;
-`;
```

Comment:
none of this really did anything, we had disablePadding on the container and then we added padding to each thing anyway?